metadata.yaml

```diff
@@ -6,3 +6,16 @@ description: |
   Charmed Ory Kratos
 summary: |
   Identity and user management system
+containers:
+  kratos:
+    resource: oci-image
+resources:
+  oci-image:
+    type: oci-image
+    description: Kratos oci-image
+    auto-fetch: true
```

Comment:
What is the use of this param? I can't find any documentation on it, only some reactive charms that are using it.